Formula/taskfile.rb

```diff
@@ -0,0 +1,44 @@
+class Taskfile < Formula
```

Comment:
Let's use `go-task` rather than taskfile, this is package inclusion, https://repology.org/project/go:task/packages
```suggestion
class GoTask < Formula
```